service/controller/resource/azuremachineconditions/conditionsubnetready.go

```diff
@@ -0,0 +1,108 @@
+package azuremachineconditions
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-11-01/network"
+	azureconditions "github.com/giantswarm/apiextensions/v3/pkg/conditions/azure"
+	"github.com/giantswarm/microerror"
+	capz "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
+	capi "sigs.k8s.io/cluster-api/api/v1alpha3"
```

Comment:
Should we run `devctl replace` to search and replace `capiv1alpha3` & similar occurrences over the whole codebase?

Reply:
Good idea :+1:

Reply:
Let's do this in a separate PR.